test/functional/repository/create_test.rb

```diff
@@ -48,4 +48,36 @@ def stub_gpg_key(gpg_key_name)
 
     assert_equal(0, run_cmd(command).exit_code)
   end
+
+  it 'shows deprecation warning on puppet content type' do
+    api_expects(:repositories, :create)
+      .with_params(
+        name: name,
+        product_id: product_id,
+        content_type: "puppet")
+
+    command = %W(repository create --organization-id #{org_id} --product-id #{product_id}
+                 --content-type puppet --name #{name})
+
+    assert('Puppet and OSTree will no longer be supported in Katello 3.16',
+           'Puppet and OSTree will no longer be supported in Katello 3.16')
```

Comment:
This assertion seems incorrect. Same with the one on line 78. It should be `assert(<something that is true>, <error message if not true>)`, like `assert(food == 'salad', 'Food isn't salad!')`

Reply:
Or `assert_equal <value1>, <value2>, message`